pycompwa/data/io/pawian.py

```diff
@@ -0,0 +1,199 @@
+"""Import Pawian data files.
```

Comment:
I thought ComPWA can do this and it is accessible via `ui`

Reply:
No, the `ui` imports a different format that has a header. In addition, the `io.pawian` module can import pawianHist files. Please have a look at the data_module notbook.

Reply:
Hmm, I just dont like this repeated code & functionality. This proves again that we should move the codebase to python

Reply:
Ok since this would be the way to do it later on in tensorwaves, this code can be nicely reused there